tests/integration/110.command.build.test.js

```diff
@@ -9,7 +9,6 @@ const { withSiteBuilder } = require('./utils/site-builder')
 
 const defaultEnvs = {
   NETLIFY_AUTH_TOKEN: 'fake-token',
-  NETLIFY_SITE_ID: 'site_id',
```

Comment:
I opted to not set it for every test here, because it does not really reflect real world scenarios.
If `NETLIFY_SITE_ID` this bug I'm fixing here never surfaces, because `@netlify/build` and `@netlify/config` both also read it.
But most users never set it probably.